utils.groovy

```diff
@@ -552,4 +552,10 @@ def get_supported_additional_arches() {
     return supported
 }
 
+// Get an emoji to use in slack notifications based on what version
+// of CoreOS we're operating on.
+def get_coreos_emoji(pipecfg) {
+    return pipecfg.streams.stable ? ':fcos:' : ':corehat:'
```

Comment:
That's a sneaky way of boiling it down to "FCOS or not". :)

I know this is just for a Slack emoji so it doesn't matter much, but I'm not a fan of the precedence it sets. I'd rather have it be configurable. One candidate would be in the `jenkins-config` configmap. So it'd be driven via `--deploy` and could default to `:coreos:`.

Also feels like we could fold this into `trySlackSend` to keep the callsites less verbose?

Reply:
> That's a sneaky way of boiling it down to "FCOS or not". :)
> 
> I know this is just for a Slack emoji so it doesn't matter much, but I'm not a fan of the precedence it sets. I'd rather have it be configurable. One candidate would be in the `jenkins-config` configmap. So it'd be driven via `--deploy` and could default to `:coreos:`.

Not a big fan of controlling it via `--deploy` because someone has to remember to add the option and also remember what the magic string sequence is. How about something in the pipecfg?

> 
> Also feels like we could fold this into `trySlackSend` to keep the callsites less verbose?

hmm. The message is pretty customized for each use though. Do you have an example of what you are suggesting?

Reply:
> Not a big fan of controlling it via `--deploy` because someone has to remember to add the option and also remember what the magic string sequence is.

It could be codified in the `HACKING.md` steps, no?

> How about something in the pipecfg?

Yeah, I thought of the `pipecfg` too, but it feels a bit weird. It's not really something you'd change very often and feels more tied to the Jenkins installation. I guess it could make sense if we start supporting different emojis per-stream? Not sure if we'd ever really want that though.

Another approach would be an annotation on the Slack secret. I still have code laying around for that if you'd like. :) But OK with `pipecfg` if you'd prefer.

> hmm. The message is pretty customized for each use though. Do you have an example of what you are suggesting?

I mean removing the prefix from all the callsites and prefixing it with the configured emoji in `trySlackSend()`. That way, we know also that all messages are consistently prefixed.

Reply:
ok I added a new commit that I think does what you were suggesting here.